api/src/main/resources/context.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "@base": "http://id.wellcomecollection.org/",
+    "@vocab": "http://wellcomecollection.org/ontologies/works/",
```

Comment:
There is no clear and widely adopted approach to this within the semantic web community. This is the best I can find from the W3C:

https://www.w3.org/blog/2016/05/https-and-the-semantic-weblinked-data/

And TimBL wants the https scheme deprecated:

https://www.w3.org/DesignIssues/Security-NotTheS.html

I have asked Data Language for the advice on the current usual practice.